ntbs-integration-tests/NotificationPages/ClinicalDetailsPageTests.cs

```diff
@@ -294,7 +318,6 @@ public async Task ValidateClinicalDetailsProperties_ReturnsErrorIfBothTreatments
 
             // Assert check just response.Content
             var result = (await response.Content.ReadAsStringAsync());
-            
             Assert.True(result.Contains(ValidationMessages.ValidTreatmentOptions));

```

Comment:
Not part of this changeset, but I believe there's an Assert.Contains which would be better suited here

Reply:
Yes indeed, thanks for spotting this :)